app.rb

```diff
@@ -0,0 +1,147 @@
+require_relative './book'
+require_relative './teacher'
+require_relative './student'
+require_relative './person'
+require_relative './rental'
+
+class App
+  def initialize()
+    @book_list = []
+    @rentals = []
+    @person = []
+  end
+
+  def menu
+    puts "Welcome to school library ! \n\n"
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts "7 - Exit\n\n"
+    gets.chomp
+  end
+
+  def menu_run
+    choice = menu.to_s
+    case choice
+    when '1'
+      list_all_books
+    when '2'
+      list_all_person
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_all_rentals_for_a_given_person_id
+    else
+      puts 'Thank you for using this App'
+      exit
+    end
+  end
+
+  def create_book
+    print 'Title: '
+    input1 = gets.chomp
+    print 'Author: '
+    input2 = gets.chomp
+    book = Book.new(input1, input2)
+    @book_list.push(book)
+    puts 'Book created successfully '
+    menu_run
+  end
+
+  def create_person
+    print "\n Do you want to create a student (1) or a teacher (2)? [Input the number]: "
+    user_input = gets.chomp
+    case user_input
+    when '1'
+      create_student
+    when '2'
+      create_teacher
```

Comment:
- [ ] As requested by the previous reviewer, Please add the `create_teacher` method. You need to create both methods to create students and teachers. In the project requirements, it's mentioned that there should be a `create_person` method that will be responsible to create students or teachers based on the user's choice.

### See the image for reuiqments

![image](https://user-images.githubusercontent.com/83803414/177016481-51c7571e-6a0a-4e82-b57e-2d62afce5f39.png)